scrapers/scrape_zg.py

```diff
@@ -78,18 +78,23 @@
 reader = csv.DictReader(StringIO(d_csv), delimiter=',')
 data = collections.defaultdict(dict)
 for row in reader:
-    if row['Typ'] == 'NA' or row['Datum'] == 'NA':
+    if row['Typ'] == 'NA' or row['Datum'] == 'NA' or row['Anzahl'] == 'NA':
         continue
     data[row['Stand']][row['Typ']] = row['Anzahl']
 days = list(data.keys())
 for day in days:
     print('-' * 10)
     dd = sc.DayData(canton='ZG', url=main_url)
     dd.datetime = day
-    dd.cases = data[day]['Fallzahl']
-    dd.hospitalized = data[day]['Hospitalisierte']
-    dd.icu = data[day]['Hospitalisierte in Intensivpflege']
-    dd.recovered = data[day]['Genesene']
-    dd.deaths = data[day]['Todesfälle']
+    if 'Fallzahl' in data[day]:
+        dd.cases = data[day]['Fallzahl']
```

Comment:
instead of checking this for every field, simply use `dd.cases = data[day].get('Fallzahl')`

if `Fallzahl` i  not there, `None` will be returned.

Reply:
updated